vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -107,3 +111,31 @@
   width: 1em;
   color: darkgrey;
 }
+
+.experiment-info {
+  flex: auto;
+  font-size: var(--lumo-font-size-s);
+  padding: var(--lumo-space-l);
+  max-width: 300px;
+}
```

Comment:
I'd suggest to be very direct in the naming here: (Since we might define an experiment-info-panel or simliar at some point later which will be overwritten) 
.experiment-info-card

Reply:
`.card .experiment-info {...}` solves this.